src/app/app.module.ts

```diff
@@ -46,9 +46,9 @@ import { StakingRouterModule } from '../staking-router-modules';
     }),
     ScheduleModule.forRoot(),
     KeyRegistryModule.forRootAsync({
-      inject: [SimpleFallbackJsonRpcBatchProvider],
-      async useFactory(provider) {
-        return { provider };
+      inject: [SimpleFallbackJsonRpcBatchProvider, ConfigService],
+      async useFactory(provider, configService) {
+        return { provider, keysBatchSize: configService?.get('KEYS_FETCH_BATCH_SIZE') };
```

Comment:
why `configService` is optional?

Reply:
another way i get TypeError: Cannot read properties of undefined (reading 'get')
but like this it works, it is possible to set keys batch via this env